src/main.js

```diff
@@ -192,7 +192,14 @@ function displayGrid() {
 }
 
 function removePoster() {
-    // use loop to remove <article> from the posterGrid and class instance from savedPosters array
+    // event.target not working - need to figure out the syntax/notation here
+    var clickedPoster = event.target;
+    // console.log(clickedPoster);
+    for (var i = 0; i < savedPosters.length; i++) {
+      if (savedPosters[i].id === clickedPoster.id)
+        savedPosters.splice(i, 1);
+        // posterGrid.HTMLCollection.remove(clickedPoster);
+    }
 }
```

Comment:
Review the Example in this mdn doc - this might help you with targeting? I'll look over this as well and see if I can figure anything out!